pkg/Library/Fields/FieldsNodeTypes.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+const category = 'UX';
+
+export const FieldsNodeTypes = {
+  StaticText: {
```

Comment:
do we need this file

Reply:
Yes, this is the original Library ... without this files, Build/Run do not work.